app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,66 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
+
+  before_action :redirect_to_pdf, if: :pdf?
+  before_action :fetch_nav, only: :show
+  before_action :proxy_content, only: :show
+
+  def index
+    @content = proxy_content
+    render :index, status: req.status
+  end
+
+  def show
+    @content = markdown? ? convert_markdown : proxy_content
+    render :show, status: req.status
```

Comment:
It does not show a blank page.  It shows a 404, with the correct status: 

![screen shot 2016-04-04 at 11 07 03](https://cloud.githubusercontent.com/assets/20921/14244864/78d3c016-fa55-11e5-93ef-3dc41bd04a04.png)